survey/survey-application/src/main/java/me/nalab/survey/application/port/out/persistence/findtarget/TargetFindPort.java

```diff
@@ -0,0 +1,20 @@
+package me.nalab.survey.application.port.out.persistence.findtarget;
+
+import java.util.Optional;
+
+import me.nalab.survey.domain.target.Target;
+
+/**
+ * targetId를 받아 Target을 반환하는 인터페이스입니다.
+ */
+public interface TargetFindPort {
+
+	/**
+	 * targetId에 해당하는 Target을 조회합니다
+	 *
+	 * @param targetId 조회할 Target의 ID
+	 * @return Optional 만약, 어떠한 targetId도 없을 경우, Optional.empty() 를 반환해야 합니다
+	 */
+	Optional<Target> findTarget(Long targetId);
```

Comment:
```suggestion
	Optional<Target> findTargetById(Long targetId);
```

이거 어떤가요? 전부 -ById를 붙이고 있는거 앝아용